### DISCUSSION TITLE
Build failed :| Step 8 script `Build ipa for distribution` exited with status code 65

### DISCUSSION THREAD

**Discussion by athmap** [codemagic-ci-cd/codemagic-docs#2329] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

<img width="1114" alt="Screenshot 2024-01-23 at 11 11 30 PM" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/146722264/b01ce66d-e5eb-4851-bf51-c536d6530102">

### Actual results

Not able to build the production ipa. Please help us to resolve. We have a release scheduled

### Build id (optional)

65aff4e38999b4b55146b7ec

**Comment by dtrdic** [comment]
Hello @athmap i can see this error inside logs
```
error: Provisioning profile "PARLO_PRODUCTION_PROFILE" doesn't support the Associated Domains capability. (in target 'parlo' from project 'parlo')
/Users/builder/clone/ios/parlo.xcodeproj: error: Provisioning profile "PARLO_PRODUCTION_PROFILE" doesn't include the com.apple.developer.associated-domains entitlement. Profile qualification is using entitlement definitions that may be out of date
```
Looks like the provisioning profile you provided doesn't have the mentioned capability, please include it into your profile and upload it to Codemagic again

**Reply by himesh-cm** [reply]
The thread was continued on intercom and resolved.

**Comment by anniewey** [comment]
@dtrdic I'm getting the same error, not sure if it's related but this error returned after upgrading node from 16.11.1 to 18.0.0 due to version compatibility error returned. Could u help to check?

```
//Build id: 65b0fec092337ea7dce83ebb

Build failed :|
Step 7 script `Build IPA for distribution` exited with status code 65
```

**Reply by himesh-cm** [reply]
Hi @anniewey, Please open a new one with the required info so we can further look into the same.

**Reply by anniewey** [reply]
@himesh-cm managed to resolve the issue after seeking help in your discord channel.
